CONTRIBUTING.md

```diff
@@ -22,15 +22,22 @@ File a single issue per problem and feature request.
 * Do not enumerate multiple bugs or feature requests in the same issue.
 * Do not add your issue as a comment to an existing issue unless it's for the identical input. Many issues look similar, but have different causes.
 
-The more information you can provide, the more likely someone will be successful reproducing the issue and finding a fix. Therefore:
+The more information you can provide, the more likely someone will be successful reproducing the issue and finding a fix. 
+
+Please include the following with each issue. 
 
-* Provide reproducible steps, what the result of the steps was, and what you would have expected.
-* A description of what you expect to happen
-* Animated GIFs
-* Code that demonstrates the issue, when providing a code snippet also include it in source and not only as an image
 * Version of VS Code
+* List of extensions that you have installed. 
```

Comment:
Suggest to add: You can easily include the list of extensions by creating the issue using the `Help>Report Issues` menu item.